blocks/init/src/Blocks/components/lists/components/lists-options.js

```diff
@@ -62,6 +64,18 @@ export const ListsOptions = (attributes) => {
 						/>
 					}
 
+					{showListsOrdered &&
+						<CustomSelect
```

Comment:
As this is less than 5 options, a `SimpleVerticalSingleSelect` would be a better option here.

There are icons for (un)ordered lists you can use 😉 
![image](https://user-images.githubusercontent.com/77000136/143472218-9a0ec601-917f-49a4-a512-325858d88a20.png)